ko/admin/config.rst

```diff
@@ -2951,4 +2967,4 @@ HA 설정은 :ref:`ha-configuration`\ 을 참고한다.
 SHARD 설정
 ==========
 
-SHARD 설정은 :ref:`default-shard-conf`\ 을 참고한다.
\ No newline at end of file
+SHARD 설정은 :ref:`default-shard-conf`\ 을 참고한다.
```

Comment:
This is because of "No newline at end of file" error.

Now it is fixed by this PR.